index.bs

```diff
@@ -10096,6 +10096,18 @@ See [[#es-DOMString]] for the specific requirements that the use of [{{TreatNull
 </div>
 
 
+<h4 id="UnenumerableOperations" extended-attribute lt="UnenumerableOperations">[UnenumerableOperations]</h4>
+
+If the [{{UnenumerableOperations}}] [=extended attribute=] appears on a [=interface=], it indicates
```

Comment:
Nit: Should there be a requirement that this extended attribute only be used on interfaces?